From 074fe469787aa65439dc63129c749517d379bb4d Mon Sep 17 00:00:00 2001
From: Stefan Saraev <stefan@saraev.ca>
Date: Mon, 3 Dec 2018 22:46:13 +0200
Subject: [PATCH] dirs

---
 libopkg/opkg_conf.h | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/libopkg/opkg_conf.h b/libopkg/opkg_conf.h
index 316c500..3f948ad 100644
--- a/libopkg/opkg_conf.h
+++ b/libopkg/opkg_conf.h
@@ -35,12 +35,12 @@ extern "C" {
 #define OPKG_CONF_TMP_DIR_SUFFIX "opkg-XXXXXX"
 #define OPKG_BACKUP_SUFFIX "-opkg.backup"
 
-#define OPKG_CONF_DEFAULT_LISTS_DIR     VARDIR "/lib/opkg/lists"
-#define OPKG_CONF_DEFAULT_INFO_DIR      VARDIR "/lib/opkg/info"
-#define OPKG_CONF_DEFAULT_STATUS_FILE   VARDIR "/lib/opkg/status"
-#define OPKG_CONF_DEFAULT_CACHE_DIR     VARDIR "/cache/opkg"
-#define OPKG_CONF_DEFAULT_CONF_FILE_DIR SYSCONFDIR "/opkg"
-#define OPKG_CONF_DEFAULT_LOCK_FILE     VARDIR "/run/opkg.lock"
+#define OPKG_CONF_DEFAULT_LISTS_DIR     "/storage/.extra/var/lib/opkg/lists"
+#define OPKG_CONF_DEFAULT_INFO_DIR      "/var/lib/opkg/info"
+#define OPKG_CONF_DEFAULT_STATUS_FILE   "/var/lib/opkg/status"
+#define OPKG_CONF_DEFAULT_CACHE_DIR     "/storage/.extra/var/cache/opkg"
+#define OPKG_CONF_DEFAULT_CONF_FILE_DIR "/etc/opkg"
+#define OPKG_CONF_DEFAULT_LOCK_FILE     "/storage/.extra/var/run/opkg.lock"
 
 /* In case the config file defines no dest */
 #define OPKG_CONF_DEFAULT_DEST_NAME "root"
-- 
2.20.1
